docs/RELEASES.md

```
@@ -0,0 +1,102 @@
+## 1. Environments & Targets
+
+| Environment | Branch | Soroban Network | Backend API | Frontend URL |
+| :--- | :--- | :--- | :--- | :--- |
+| **Local** | `N/A` | `standalone` | `localhost:3001` | `localhost:3000` |
+| **Preview** | PR branches | `N/A` (Tests) | `api-preview.vaultquest.app` | Vercel/Netlify Preview |
+| **Staging** | `develop` | `testnet` | `api-staging.vaultquest.app` | `staging.vaultquest.app` |
+| **Production** | `main` | `public` | `api.vaultquest.app` | `vaultquest.app` |
+
+## 2. Artifact Propagation
+
+### Soroban Contracts
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)